### DISCUSSION TITLE
有哪些网盘可以用rclone挂载再由同步工具（比如verysync）同步大量本地文件作备份用？

### DISCUSSION THREAD

**Discussion by AspirantH** [AlistGo/alist#4846] [General]
原先123盘一直用的很好，近一个月不断抽风
阿里云盘是超级会员，但是我发现rclone挂载阿里云盘OPEN后可以读取，因为emby一直都可以正常看里面的视频，但就是原先在123盘上用的同样的方法在阿里云盘open上就不行，表现在实际是备份到本地的磁盘空间里了，即使rclone挂在本地的文件路径就是网盘路径，比如/alist/阿里云盘/ABC目录，但实际上却是保存在本地服务器上的，网盘里根本没有所要备份的文件。请问大家有什么方法可以实现这种需求吗？

**Comment by AspirantH** [comment]
![微信截图_20230725005037](https://github.com/alist-org/alist/assets/93640658/f19fe4b8-25fc-46bd-8760-4c04cd3649c2)
![微信截图_20230725005103](https://github.com/alist-org/alist/assets/93640658/6accc1f4-d52d-4bf7-8e18-09aa78fcd792)
包括蓝奏也是一样，不知道我的表述大家是否明白意思

**Comment by AspirantH** [comment]
看图里看上去是一直在同步文件，实际上挂载的网盘中却没有任何文件

**Reply by cyay10** [reply]
你解决了吗。

**Reply by XingHaiHuShen** [reply]
@cyay10 您解决了么？我也遇到了相同的问题

**Comment by XingHaiHuShen** [comment]
请问您是否解决了这个问题？我现在也遇到了同样的困扰
